internal/cli/atlas/backup/snapshots_list.go

```diff
@@ -38,14 +38,18 @@ func (opts *SnapshotsListOpts) initStore() error {
 	return err
 }
 
+var listTemplate = `ID	TYPE	STATUS	CREATE	EXPIRE{{range .Results}}
```

Comment:
[nit-grammar] similar to above, missing `D` and `S` but is standard for dates to include an `AT`
```suggestion
var listTemplate = `ID	TYPE	STATUS	CREATED AT	EXPIRES AT{{range .Results}}
```